x-pack/plugins/observability_solution/slo/public/context/plugin_context.tsx

```diff
@@ -24,6 +24,6 @@ export interface OverviewEmbeddableContextValue {
   observabilityRuleTypeRegistry: ObservabilityRuleTypeRegistry;
 }
 
-export const OverviewEmbeddableContext = createContext<OverviewEmbeddableContextValue | null>(null);
-
-export const PluginContext = createContext<PluginContextValue | null>(null);
+export const PluginContext = createContext(
+  {} as PluginContextValue | OverviewEmbeddableContextValue
```

Comment:
shouldn't this always be `PluginContextValue` ?

Reply:
On a dashboard context it should be `OverviewEmbeddableContextValue` otherwise `PluginContextValue`. Let me push the refactored code. I disabled an eslint rule though to make it work, so I'd like your thoughts on this.